index.html

```diff
@@ -54,7 +54,170 @@ <h2 id="name_title">I’m Chima Glad to see you!</h2>
                     </ul>
                 </nav>
             </div>                        
-        </section>        
+        </section>      
+        <section id="works">
+            <div class="card">
+                <a href="#" class="project_cta_link">
+                    <img src="assets/images/SnapshootPortfolio.png" alt="Project Name" class="snapshot_portfolio">
+                </a>
+                <div class="project_description">
+                    <div class="project_title">Tonic</div>
+                    <div class="project_summary">
+                        <ul class="summary_list">
+                            <li class="summary_list_item no_disc">canopy</li>
+                            <li class="summary_list_item"> back end dev</li>
+                            <li class="summary_list_item">2015</li>
+                        </ul>
+                    </div>
+                    <p class="project_text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
<img width="810" alt="4" src="https://user-images.githubusercontent.com/77843050/212476011-7efae91f-39f8-45a7-887e-cd1c781c6196.png">

- [ ] Not much to say, all is clear here, your technologies color is not the same as the one on the left, please change that
- [ ] The space after your `see project` button is considerably wide 😄 , what do you think if you make a few modifications?
- [ ] After that you will have to update the space between your cards though, as highlighted in red,